docs/meshcloud.policies.md

```diff
@@ -5,11 +5,51 @@ title: meshPolicies
 
 ## Introduction: what are meshPolicies?
 
-A meshPolicy describes a restriction between two [meshObjects](meshcloud.index.md#introduction) that are globally defined by your organization. Each meshPolicy describes which [tag](meshcloud.metadata-tags.md) values have to be present on both meshObjects, in order to comply with the meshPolicy. meshPolicies are enforced in various places when attaching meshObjects. How meshPolicies can be configured is described [here](administration.mesh-policies.md).
+A meshPolicy describes a restriction between two meshObjects. You can only define a meshPolicy on certain meshObjects by using their defined tags. These particular meshObjects are called meshSubjects. For ease of use, we restrict the combinations of meshSubjects which do not make sense. Each meshPolicy describes which [tag](meshcloud.metadata-tags.md) values have to be present on both meshSubjects, in order to comply with the meshPolicy.
+
+> **meshCustomer**, **meshProject**, **meshUser/Group** and **meshLandingZone** are meshSubjects
+
+You can find the authoritative meshSubject on the left side and the affected meshSubject on the right side. Each meshPolicy describes which [tag](meshcloud.metadata-tags.md) values have to be present on both meshSubjects, in order to comply with the meshPolicy. To decide if a meshPolicy is compliant it is necessary to select an evaluation strategy how you want to evaluate the authoritative meshSubject against the affected meshSubject. meshPolicies are enforced in various places when attaching meshSubjects. How meshPolicies can be configured is described [here](administration.mesh-policies.md).
+
+> meshPolicies will only be enforced for meshCustomers. meshPartners are excluded.
+
+### Authoritative and affected meshSubject
+
+The terms authoritative and affected meshSubject shall make clear in which direction the meshPolicy will be evaluated. In general the authoritative meshSubject restricts the affected meshSubject depending on the selected evaluation strategy.
+
+We've a limited selection of meshSubject combinations. Valid combinations are
+
+- meshCustomer -> meshUser/Group
+- meshCustomer -> meshProject
+- meshProject -> meshUser/Group
+- meshProject -> meshLandingZone
+
+### meshPolicy evaluation strategy
+
+A meshPolicy evaluation stategy describes a strategy how authoritative and affected meshSubjects shall be evaluated in context of a meshPolicy. There exists two different selectable evaluation options `Subset` and `Intersection`.
+
+#### Subset
```

Comment:
It would be great to have an example of this. It is too theoretical at the moment.

Reply:
We'll improve the docs afterwards. This is not the final state.